web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknad/SøknadRoutes.kt

```diff
@@ -48,27 +53,69 @@ internal fun Route.søknadRoutes(
             )
         }
     }
+
+    authorize(Brukerrolle.Saksbehandler) {
+        post("$søknadPath/{søknadId}/lukk") {
+            call.withSøknadId { søknadId ->
+                Either.catch { deserialize<Søknad.LukketSøknadBody>(call) }.fold(
```

Comment:
Fordi saksbehandler skal legge in dato for brev, lagde jeg en body for kallet, så slipper vi å ha dato som parameter. + vi skal også ha et par andre ting fra det jeg kan si.